packages/dso-toolkit/src/components/button/button.mixins.scss

```diff
@@ -178,18 +178,18 @@
       &::after {
         content: "";
         display: inline-block;
-        margin-left: units.$u1;
+        margin-inline-start: units.$u1;
       }
     }
   }
 }
 
 @mixin block($sibling-selector) {
   display: block;
-  width: 100%;
+  inline-size: 100%;
 
   & + #{$sibling-selector} {
-    margin-top: 5px; // Vertically space out multiple block buttons
+    margin-block-start: 5px; // Vertically space out multiple block buttons
```

Comment:
Commentaar gaat nog uit van physical properties :)

Reply:
Dat mag blijven staan. Tenzij we naar klassiek Chinees gaan, blijft verticaal verticaal :)

Reply:
comment is geheel verwijderd